documentation/modules/ref-operator-cluster.adoc

```diff
@@ -182,7 +183,31 @@ If the `ConfigMap` is missing when the Cluster Operator is deployed, the default
 If the `ConfigMap` is accidentally deleted after the Cluster Operator is deployed, the most recently loaded logging configuration is used.
 Create a new `ConfigMap` to load a new logging configuration.
 
-NOTE: Do not remove the monitorInterval option from the ConfigMap.
+NOTE: Do not remove the `monitorInterval` option from the ConfigMap.
+
+== Restricting Cluster Operator access with network policy
+
+The Cluster Operator can run in the same namespace as the resources it manages, or in a separate namespace.
+The `STRIMZI_OPERATOR_NAMESPACE` environment variable is configured by default, and uses the Kubernetes Downward API to find which namespace the Cluster Operator is running in.
+If the Cluster Operator is running in the same namespace, only local access is required, and allowed by Strimzi.
+
+If the Cluster Operator is running in a separate namespace to the resources it manages, any namespace in the Kubernetes cluster is allowed access to the Cluster Operator unless network policy is configured.
+Use the optional `STRIMZI_OPERATOR_NAMESPACE_LABELS` environment variable to establish network policy for the Cluster Operator using namespace labels.
+By adding namespace labels, access to the Cluster Operator is restricted to the namespaces specified.
+
+.Network policy configured for the Cluster Operator deployment
+[source,yaml,options="nowrap"]
+----
+#...
+env:
+  - name: STRIMZI_NAMESPACE
+    valueFrom:
+      fieldRef:
+        fieldPath: metadata.namespace
```

Comment:
I think this might confuse readers to understand the relation ship between these two.

```suggestion
  # ...
```